src/main/kotlin/com/launchdarkly/intellij/FlagStore.kt

```diff
@@ -56,6 +57,11 @@ class FlagStore(private var project: Project) {
             envList = listOf(settings.environment)
             return getFlags.getFeatureFlags(ldProject, envList, true, null, null, null, null, null, null)
         } catch (err: Exception) {
+            val notifier = GeneralNotifier()
+            notifier.notify(
+                project,
+                "Error retrieve flags: ${err.message}"
+            )
             println(err)
```

Comment:
personal preference - I think we should always print the error before notifying _just in case_ there's a notify err, i'd still want the caught err logged

Reply:
sounds good to me, I'll get the spots updated